trollmoves/server.py

```diff
@@ -657,11 +657,14 @@ def copy(self):
         LOGGER.debug('dest path %s ', os.path.dirname(self.destination.path))
         LOGGER.debug('origin %s ', self.origin)
 
-        scp = SCPClient(ssh.get_transport())
-        scp.put(self.origin, self.destination.path)
-
-        scp.close()
-        ssh.close()
+        try:
+            scp = SCPClient(ssh.get_transport())
+            scp.put(self.origin, self.destination.path)
+        except Exception as e:
+            LOGGER.error("Something went wrong with scp: " + str(e))
+        finally:
+            scp.close()
```

Comment:
if the `SCPClient` call fails, the `scp` object isn't creates, and the subsequent call to `close` will crash...